virtwho/virt/ahv/ahv_interface.py

```diff
@@ -514,14 +514,22 @@ def get_vms_uuid(self, version):
         self._logger.info("Getting the list of available vms")
         is_pc = True if version == 'v3' else False
         vm_uuid_list = []
-        length = 0
+        length = ahv_constants.NUM_OF_REQUESTED_VMS
+        initial_offset = 0
         offset = 0
         total_matches = 0
         count = 1
         current = 0
         (url, cmd_method) = self.get_diff_ver_url_and_method(
             cmd_key='list_vms', intf_version=version)
-        res = self.make_rest_call(method=cmd_method, uri=url)
+        if is_pc is True:
+            body = {
+                'length': length,
+                'offset': initial_offset
+            }
+            res = self.make_rest_call(method=cmd_method, uri=url, json=body)
+        else:
+            res = self.make_rest_call(method=cmd_method, uri=url)
```

Comment:
I think this can be simplified a bit, removing the duplicated `make_rest_call()` call & args, by composing its arguments as kwargs.